specification/wse/src/main/scripts/org/kaazing/specification/wse/data/binary.as.escaped.text/echo.payload.length.127/request.rpt

```diff
@@ -1,5 +1,5 @@
 property sequence ${wse:randomInt(100)}
```

Comment:
This test is ignored and will be fixed assuming the interpretation of the spec above is correct